src/avro/src/error.rs

```diff
@@ -91,7 +91,11 @@ pub enum DecodeError {
     IntDecodeOverflow,
     BadJson {
         category: serde_json::error::Category,
-        original_bytes: Option<Vec<u8>>,
+        /// A string representation of what we attempted to decode.
+        /// Ideally the original bytes,
+        /// but might be a re-serialization of a deserialized value,
+        /// if we no longer have access to the original.
+        string: Vec<u8>,
```

Comment:
nit: this isn't actually a string! Maybe `encoded_repr` or something like that? Or make it a real `String`?